cli/src/commands/create/dataset.rs

```diff
@@ -22,9 +22,9 @@ pub struct CreateDatasetArgs {
     /// Set the description of the new dataset
     description: Option<String>,
 
-    #[structopt(long = "has-sentiment")]
+    #[structopt(long = "has-sentiment", parse(try_from_str), default_value = "false")]
     /// Enable sentiment prediction for the dataset
-    has_sentiment: Option<bool>,
+    has_sentiment: bool,
```

Comment:
Likewise, these types should accurately reflect the user's input. I think we can leave the type how it is, to capture whether the user specified the flag, or left it as the default value.